entries/g/gatech.edu.json

```diff
@@ -1,18 +1,20 @@
 {
   "Georgia Institute of Technology": {
     "domain": "gatech.edu",
-    "url": "https://www.gatech.edu",
     "tfa": [
       "call",
-      "custom-software",
-      "custom-hardware"
+      "sms",
+      "custom-software"
```

Comment:
They still support [hardware tokens](https://gatech.service-now.com/home?id=kb_article_view&sysparm_article=KB0027601). It's just buried in their help center, but there are screenshots in the [main help center article](https://gatech.service-now.com/home?id=kb_article_view&sysparm_article=KB0027575).

Reply:
Hmm, they state that Duo hardware tokens are supported but note that only "Only Blue Duo Hardware Tokens are supported".
I've never heard of a blue D100 Token but apparently [Wellesley College](https://www.wellesley.edu/lts/duo/duo-usb-token) refers to the Yubikey FIDO key as a Blue Duo Hardware Token. It could be that the same is true for GA Tech :thinking:

Reply:
Odd

Reply:
Added U2F method to the entry.